python/lsst/ts/m2/csc.py

```diff
@@ -234,16 +234,23 @@ async def telemetry_loop(self):
 
                 self.tel_powerStatus.put()
 
+                temperature_change = np.random.normal(
+                        scale=self.temperature_rms, size=self.n_ring_temperatures
+                    )
+                if np.any(np.array(self.tel_temperature.data.ring) > 20):
```

Comment:
Instead of having `20` and `0` hard coded here, create `self.max_temperature` and `self.min_temperature` attributes in the `__init__` class.